RadixWallet/Core/SharedModels/PreviewValues/Profile/Account+PreviewValues.swift

```diff
@@ -1,47 +1,13 @@
 #if DEBUG
 
-extension Profile.Network.Account {
-	public static let previewValue0: Self = try! Self(
```

Comment:
Sargon provides sample values for ALL kinds, and TWO of them per type, called `sample` and `sampleOther`. We should remove all those `previewValue`s...